lib/CtrlNest/Controller/Auth.pm

```diff
@@ -38,28 +71,47 @@ sub lockscreen {
 
   # Render template "auth/lockscreen.html.ep"
   return $self->render(
-    layout      => 'auth',
-    title       => 'Lockscreen',
-    page_layout => 'lockscreen',
-    username    => $username
+    layout   => 'auth',
+    title    => 'Lockscreen',
+    username => $username,
+
+    # css class for the body
+    auth_layout => 'hold-transition lockscreen'
```

Comment:
Let's look at the AdminLTE documentation and try to find a solution to get rid of this variable.